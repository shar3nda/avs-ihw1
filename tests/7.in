-6 -9 -5 -6 -1 -2 -8 -7 -6 -8