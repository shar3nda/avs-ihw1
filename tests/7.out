-2 -1 -1 3 9 5 0 8